ground/src/test/java/com/google/android/ground/ui/map/gms/mog/MogSourceTest.kt

```diff
@@ -24,24 +24,24 @@ import org.mockito.junit.MockitoJUnitRunner
 
 @RunWith(MockitoJUnitRunner::class)
 class MogSourceTest {
-  private val mogSource = MogSource("url/{x}/{y}.tif", 5..7)
+  private val mogSource = MogSource( 5..7, "url/{x}/{y}.tif")
 
   @Test
-  fun getMogUrl_whenZoomIsLessThanMinZoom_throwsError() {
+  fun getMogPath_whenZoomIsLessThanMinZoom_throwsError() {
     assertThrows(IllegalStateException::class.java) {
-      mogSource.getMogUrl(TileCoordinates(250, 500, 2))
+      mogSource.getMogPath(TileCoordinates(250, 500, 2))
     }
   }
 
   @Test
-  fun getMogUrl_whenZoomIsEqualToMinZoom_returnsMogUrlWithMinZoom() {
-    assertThat(mogSource.getMogUrl(TileCoordinates(250, 500, 5))).isEqualTo("url/250/500.tif")
+  fun getMogPath_whenZoomIsEqualToMinZoom_returnsMogUrlWithMinZoom() {
```

Comment:
Consider renaming tests in this class using backtick delimiting, for ex:
fun `getMogPath returns low res MOG url when zoom == min zoom`()

Reply:
Thank you! I saw this suggestion in one of the Kotlin books I was reading. Done.